tests/test_zpcore.py

```diff
@@ -137,6 +150,22 @@ def test_create_new_system_raises_valueerror(zos, simple_system, connection_mode
         zos.create_new_system()
 
 
+@pytest.fixture()
+@pytest.mark.require_mode("standalone")
```

Comment:
This marker appears not to work and can be removed, see https://docs.pytest.org/en/latest/how-to/mark.html